httpclient/httpclient.go

```diff
@@ -50,6 +50,9 @@ type Opts struct {
 
 	// MetricsCollector is a metrics collector.
 	MetricsCollector MetricsCollector
+
+	// ClassifyRequest does request classification, producing non-parameterized summary for given request.
```

Comment:
Please write few words that `ClassifyRequest` is used for `MetricsRoundTripper` for now.